src/SprykerEco/Yves/Computop/Handler/PostPlace/ComputopPayPalExpressPaymentHandler.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Computop\Handler\PostPlace;
+
+use Generated\Shared\Transfer\ComputopPayPalExpressPaymentTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Shared\Kernel\Transfer\AbstractTransfer;
+
+class ComputopPayPalExpressPaymentHandler extends AbstractPostPlacePaymentHandler
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Spryker\Shared\Kernel\Transfer\AbstractTransfer $responseTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    protected function addPaymentToQuote(QuoteTransfer $quoteTransfer, AbstractTransfer $responseTransfer): QuoteTransfer
+    {
+        if ($quoteTransfer->getPayment()->getComputopPayPalExpress() === null) {
+            $computopTransfer = new ComputopPayPalExpressPaymentTransfer();
+            $quoteTransfer->getPayment()->setComputopPayPalExpress($computopTransfer);
+        }
+        /** @var \Generated\Shared\Transfer\ComputopPayPalExpressInitResponseTransfer $responseTransfer */
+        $quoteTransfer->getPayment()->getComputopPayPalExpress()->setPayPalExpressInitResponse(
+            $responseTransfer
+        );
+
+        return $quoteTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    protected function saveInitResponse(QuoteTransfer $quoteTransfer): QuoteTransfer
+    {
+        return $this->computopClient->savePayPalExpressInitResponse($quoteTransfer);
```

Comment:
please do not wrap one line methods into other methods

Reply:
Is it even used?